Std/Data/AssocList.lean

```diff
@@ -43,6 +44,18 @@ def isEmpty : AssocList α β → Bool
 @[simp] theorem isEmpty_eq (l : AssocList α β) : isEmpty l = l.toList.isEmpty := by
   cases l <;> simp [*, isEmpty, List.isEmpty]
 
+/-- The number of entries in an `AssocList`. -/
+def size (L : AssocList α β) : Nat :=
+  match L with
```

Comment:
```suggestion
def length : AssocList α β → Nat
```
The naming difference between `List` and other data structures is unfortunate, but `AssocList` takes its API mainly from `List` so I think it should use `length` here. Or maybe one could say that `length` is for the O(n) operation and `size` is when it is O(1) because of some kind of caching? In which case this should also be `length`.